promisasu/Portal
# Update Node Modules

**Taiga User Story**
None

**Taiga Task(s)**
None

**What did you change?**
Updated eslint, eslint-plugin-ava, hapi, pm2, remark-lint, and sequelize.
Resolves issue with atom-eslint throwing an error while writing unit tests.

**How can we test?**
`npm install`
restart computer
`npm start`
portal and api should work the same as before :smile:
